contracts/gxc.htlc/src/gxc.htlc.cpp

```diff
@@ -23,7 +21,7 @@ void htlc_contract::newcontract(name owner, string contract_name, std::variant<n
       lck.timelock = timelock;
    });
 
-   transfer_action("gxc.token"_n, {{_self, "active"_n}}).send(owner, _self, value, "");
+   transfer_action("gxc.token"_n, {{_self, "active"_n}}).send(owner, _self, value, "CREATED FROM " + owner.to_string() + (contract_name.size() ? ", " : "") + contract_name);
```

Comment:
```suggestion
   transfer_action("gxc.token"_n, {{_self, "active"_n}}).send(owner, _self, value, "CREATED BY " + owner.to_string() + (contract_name.size() ? ", " : "") + contract_name);
```
What about `by` instead of `from`?